test/spec/resolver/node-resolver-spec.js

```diff
@@ -11,125 +11,233 @@ describe('resolver/node-resolver', function() {
 
     const resolver = createResolver(function(path) {
 
-      // mimic local package look-up
+      // mock local package.json resolving
       if (path === './package.json') {
         return {
           name: 'bpmnlint-plugin-local'
         };
       }
 
-      if (path.includes('rules/non-existing')) {
-        throw new Error('not found');
+      /**
```

Comment:
Can we split this up in refactoring (test re-structuring) and test covering the new feature?

Reply:
Yes!